torch/csrc/jit/codegen/cuda/utils.cpp

```diff
@@ -101,8 +110,9 @@ auto parseDebugDumpOptions() {
             "\tcuda_to_file, debug_info, launch_param, segmented_fusion, fusion_args,\n",
             "\tkernel_args, dump_eff_bandwidth, draw_segmented_fusion,\n",
             "\tscheduler_params, parallel_dimensions, buffer_reuse_verbose,\n",
-            "\tptxas_verbose, halo, segmenter_logging, perf_debug_verbose\n",
-            "\ttransform_propagator, inline_propagator\n");
+            "\tptxas_verbose, halo, segmenter_logging, perf_debug_verbose,\n",
+            "\tpython_definition, python_frontend_debug, ttransform_propagator,\n",
```

Comment:
nit:
```suggestion
            "\tpython_definition, python_frontend_debug, transform_propagator,\n",
```

Reply:
Also, `fusion_ir_presched` is missing?

Reply:
Good question!  I will check.

Reply:
Fixed both the typo and putting in `fusion_ir_presched`.